test/basic.ml

```diff
@@ -158,6 +158,43 @@ let rectangle () =
   let r = Rtree.Rectangle.merge_many [ r1; r2 ] in
   assert (r = r3)
 
+  (* Testing iter function *)
+let test_iter () =
+  let module R =
+    Rtree.Make
+      (Rtree.Rectangle)
+      (struct
+        type t = int
+
+        let t = Repr.float
+
+        let envelope { Rtree.Rectangle.x_min = x1; y_min = y1; x_max = x2; y_max = y2 } =
```

Comment:
The function `envelope` needs to go from the type `t` to a rectangle. It might be easier to copy something from above that uses a line for this test ?